libraries/src/Helper/ModuleHelper.php

```diff
@@ -451,7 +451,7 @@ public static function getModuleList()
 	public static function cleanModuleList($modules)
 	{
 		// Apply negative selections and eliminate duplicates
-		$Itemid = \JFactory::getApplication()->input->getInt('Itemid');
+		$Itemid = (int) \JFactory::getApplication()->input->getInt('Itemid', 0);
 		$negId = $Itemid ? -(int) $Itemid : false;
```

Comment:
```suggestion
		$negId = $Itemid ? -$Itemid : false;
```
One case should be enough